### PR TITLE
EIP1-2471: EIP1-2780: Add supportingInformationFormat to SendApplicationToPrintMessage

### PR DESCRIPTION
**Pull request by adacayi**
This PR is to add the supporting information format to `SendApplicationToPrintMessage`, so the value can be received from VCA and later persisted and sent to print provider in the agreed format.

### CODE REVIEW

File: src/main/kotlin/uk/gov/dluhc/printapi/config/JacksonConfiguration.kt
Comment (adacayi) [inline src/main/kotlin/uk/gov/dluhc/printapi/config/JacksonConfiguration.kt:24]: I thought this was already added as part of the development process for extending the ero response to include new contact fields so that the application won't fail. 

Since we are happy to configure the deserializer to not fail for unknown fields in other apis, I think it will ease our jobs, if we have it here as well.

File: src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml
Comment (NathanRussellValtech) [inline src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml:112]: Can we add this new property to the `required` array please?
We need it to be a mandatory property in the message (though see my other comment about chicken and egg 😁 )

File: src/main/kotlin/uk/gov/dluhc/printapi/config/JacksonConfiguration.kt
Comment (Neil-Massey) [reply src/main/kotlin/uk/gov/dluhc/printapi/config/JacksonConfiguration.kt:24]: So did I... https://github.com/cabinetoffice/eip-ero-print-api/pull/93/files

**Review by Neil-Massey — APPROVED** [review]
LGTM

**Review by NathanRussellValtech — APPROVED** [review]
LGTM; (assuming you can add `supportingInformationFormat` as a required property 👍 )

But .... you won't be able to merge this yet - we have a chicken and egg situation! 🙄 
What you'll need to do it take this spec (1.8.0) including the required `supportingInformationFormat` property, and bring it into VCA, and then update the VCA code that creates this message to map in the `supportingInformationFormat` field
Once that change is merged and deployed we can then merge and deploy this change.

Or .... you merge it as you have developed (where `supportingInformationFormat` is not required); then do the VCA change, then come back to this and make the field mandatory and up-version it again :( 

Either way is a bit messy :(

File: src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml
Comment (adacayi) [reply src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml:112]: @NathanRussellValtech , maybe for this change I only change the Jackson Configuration to allow unknown fields and then update VCA message and the spec to include the new field and then update `print-api` with the new spec as well.

**Pull request merged**
